console/src/app/pages/users/user-detail/auth-user-detail/resend-email-dialog/resend-email-dialog.component.html

```diff
@@ -0,0 +1,19 @@
+<h1 mat-dialog-title>
+    <span class="title">{{'USER.SENDEMAILDIALOG.TITLE' | translate}} {{data?.number}}</span>
+</h1>
+<p class="desc">{{'USER.SENDEMAILDIALOG.DESCRIPTION' | translate}}</p>
+<div mat-dialog-content>
+    <mat-form-field class="formfield">
+        <mat-label>{{ 'USER.SENDEMAILDIALOG.NEWEMAIL' | translate }}</mat-label>
+        <input matInput [(ngModel)]="email" />
+    </mat-form-field>
+</div>
+<div mat-dialog-actions class="action">
+    <button cdkFocusInitial color="primary" mat-button class="ok-button" (click)="closeDialog()">
```

Comment:
should'n this be secondary?

Reply:
youre right. ive removed initial focus on cancel